src/ExpressionEvaluator/Core/Test/ResultProvider/Debugger/Engine/DkmInspectionContext.cs

```diff
@@ -38,10 +38,10 @@ internal DkmInspectionContext(IDkmClrFormatter formatter, DkmEvaluationFlags eva
         //     '16' and '10'.
         public readonly uint Radix;
 
-        public string GetTypeName(DkmClrType clrType, ReadOnlyCollection<string> formatSpecifiers, DkmClrCustomTypeInfo CustomTypeInfo)
+        public string GetTypeName(DkmClrType ClrType, DkmClrCustomTypeInfo DeclaredTypeInfo, ReadOnlyCollection<string> FormatSpecifiers)
```

Comment:
If these aren't going to match (ClrType/DeclaredTypeInfo), you might as well stick with CustomTypeInfo.